### PR TITLE
Add hackcon.online CTF writups

### PR DESCRIPTION
**Pull request by aenniw**
Signed-off-by: Martin Mihálek <aenniw@gmail.com>

### CODE REVIEW

File: hackcon.online/invisible-programming/hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#include		  	  <iostream> 
+using 		 	  namespace std;  
+int main			  	 () 
+{		 	 		
+	cout				 		<<"The flag is d4rk{You_thought_this_will_be that_much_easy}c0de\n"; 
+	return 	  		  0; 
+		  		} 
+			 	  
+	  			
+
```
Comment (fabrymarko) [inline hackcon.online/invisible-programming/hello_world.cpp]: WTF?

File: hackcon.online/jorge-luis-borges/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+cat ./file2.txt | head -n 1
+cat ./file2.txt | sed 's/!/!\n\nx = 0, y = 0, rule = B3\/S23\n/g'
+```
+
+Import and check candidates in https://copy.sh/life/
+
+![life.png](./life.png ':ignore')
+
+```bash
+echo 'YmFiZWw=' | base64 -d     # google to https://libraryofbabel.info
```
Comment (fabrymarko) [inline hackcon.online/jorge-luis-borges/README.md]: even bigger WTF?

File: hackcon.online/wierd-text/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+---
+
+#### Solution:
+
+https://en.wikipedia.org/wiki/Esoteric_programming_language
+
+```bash
+# branfuck          | malbolge | reverse chars
+bf ./mysterious.txt | malbolge | xxd -r -p | tac -r -s 'x\|[^x]'
```
Comment (fabrymarko) [inline hackcon.online/wierd-text/README.md]: bf i know, bot WTF is the rest?

File: hackcon.online/baby-bOf/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+   0x000000000040072e <+103>:   call   0x4005d0 <setvbuf@plt>
+   0x0000000000400733 <+108>:   mov    DWORD PTR [rbp-0x4],0xcafebabe
+   0x000000000040073a <+115>:   mov    QWORD PTR [rbp-0xe],0x0
+   0x0000000000400742 <+123>:   mov    WORD PTR [rbp-0x6],0x0
+   0x0000000000400748 <+129>:   mov    rdx,QWORD PTR [rip+0x200921]        # 0x601070 <stdin@@GLIBC_2.2.5>
+   0x000000000040074f <+136>:   lea    rax,[rbp-0xe]
+   0x0000000000400753 <+140>:   mov    esi,0x100
+   0x0000000000400758 <+145>:   mov    rdi,rax
+   0x000000000040075b <+148>:   call   0x4005c0 <fgets@plt>
+   0x0000000000400760 <+153>:   cmp    DWORD PTR [rbp-0x4],0xdeadbeef
```
Comment (fabrymarko) [inline hackcon.online/baby-bOf/README.md]: add comment after # so stupid people know that this is the place where the magic happens

File: hackcon.online/babypwn/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+```console
+(gdb) print win
+$2 = {<text variable, no debug info>} 0x400831 <win>
+(gdb) b *0x004009a2
+(gdb) r
+(gdb) x/x $rax
+0x7fffffffda20:	0x00000000
+(gdb) x/x $rbp-0x40
+0x7fffffffda10:	0x00400817
+(gdb) p ($rbp-0x40) - $rax
```
Comment (fabrymarko) [inline hackcon.online/babypwn/README.md]: also comments of magic spots would be nice

File: hackcon.online/invisible-programming/hello_world.cpp (hunk 1 above)
Comment (aenniw) [reply hackcon.online/invisible-programming/hello_world.cpp]: tab -> 1
space -> 0

File: hackcon.online/wierd-text/README.md (hunk 3 above)
Comment (aenniw) [reply hackcon.online/wierd-text/README.md]: malbogne esoteric language and rest is just hex to ascii and string reverse

File: hackcon.online/babypwn/README.md (hunk 5 above)
Comment (aenniw) [reply hackcon.online/babypwn/README.md]: Will add to img ...

File: hackcon.online/invisible-programming/hello_world.cpp (hunk 1 above)
Comment (fabrymarko) [reply hackcon.online/invisible-programming/hello_world.cpp]: comment that in code please, ideally in sentences describing how you solved this diabolical plot.

File: hackcon.online/wierd-text/README.md (hunk 3 above)
Comment (fabrymarko) [reply hackcon.online/wierd-text/README.md]: in comment add link to wiki / codechef decryptor maybe

File: hackcon.online/jorge-luis-borges/README.md (hunk 2 above)
Comment (fabrymarko) [reply hackcon.online/jorge-luis-borges/README.md]: what is this library of babel? it looks like math to me.

https://www.youtube.com/watch?v=JYqjcHYTQgQ

File: hackcon.online/baby-bOf/README.md (hunk 4 above)
Comment (aenniw) [reply hackcon.online/baby-bOf/README.md]: Updated in img

File: hackcon.online/wierd-text/README.md (hunk 3 above)
Comment (aenniw) [reply hackcon.online/wierd-text/README.md]: added links, cyberschef doesn't support esoteric langs

File: hackcon.online/jorge-luis-borges/README.md (hunk 2 above)
Comment (aenniw) [reply hackcon.online/jorge-luis-borges/README.md]: `'YmFiZWw='` is shown in `life.png`and babel lib is just library where you can find things via codes... it's something you need to nagite through

File: hackcon.online/invisible-programming/hello_world.cpp (hunk 1 above)
Comment (aenniw) [reply hackcon.online/invisible-programming/hello_world.cpp]: Added description

**Pull request merged**